department-of-veterans-affairs/abd-vro
# MCP-2108 V2 Health Assessment

## What was the problem?
Keep v1/full-health-data-assessment endpoint and create a v2/health-data-assessment

Associated tickets or Slack threads:

- [MCP-2108](https://amida.atlassian.net/browse/MCP-2108)

## How does this fix it?[^1]


## How to test this PR
- Step 1 run the app
- Step 2 try out v2/health-data-assessment (will need a JWT bearer)
- Step 3 verify v1/full-health-data-assessment still works


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
